about.md

```diff
@@ -4,20 +4,66 @@ title: About
 permalink: /about/
 ---
 
-## Short History of Open ONI
+<p>Open Online Newspaper Initiative (Open ONI) is a community-maintained project to make historic American newspapers browsable and searchable on the web. We aim to enable individual state and library-based newspaper collections to more easily provide these core capabilities, as well as develop new common features and improvements.</p>
 
-- 2007-2009 - Library of Congress Developed “chronam” to support the National Digital Newspaper Program (NDNP) sponsored by the National Endowment of the Humanities (NEH)
-- 2011 - Library of Congress released “chronam” open source and presented at code4lib, released on SourceForge
-- 2015 - Community sprint to fork chronam into “Open ONI” and plan features
-- 2017 - Official release of Open ONI
-- 2017 - v.0.6.0 released with better support for IIIF and Universal Viewer
+<p>There are a number of sites using Open ONI in production. Links to these
+websites and their source code are available on
+<a href="https://github.com/open-oni/open-oni/wiki/Sites-Using-Open-ONI" rel="nofollow">
+the Open ONI wiki</a>. If you are using Open ONI to power your site, please let
```

Comment:
Can we change this link's text to fit the target better?
Maybe more like `<a href="...">Sites Using Open ONI</a> on our wiki.`